ansible/host_vars/a-hel-fi.m.voidlinux.org.yml

```diff
@@ -49,6 +49,9 @@ nomad_host_volumes:
   - name: ccache
     path: /hostdir/ccache
     read_only: true
+  - name: popcorn_data
+    path: /nomad/popcorn_data
+    read_only: false
```

Comment:
do we want to keep popcorn on `a-hel-fi`? I don't see any reason it needs to stay there